src/journal/journald-server.h

```diff
@@ -51,6 +51,17 @@ typedef enum SplitMode {
         _SPLIT_INVALID = -1
 } SplitMode;
 
+typedef struct JournalInfo {
+        const char *name;
+        const char *path;
+
+        JournalMetrics metrics;
+
+        uint64_t cached_space_available;
+        uint64_t cached_space_limit;
+        usec_t   cached_space_timestamp;
+} JournalInfo;
```

Comment:
I figure it makes sense to add a structure for this, but could we maybe name this "JournalStorageInfo" or just "JournalStorage" or so? i.e. "info" alone is so overly unprecise. Given that we instantiate this for the two storage locations each, I think it should be called "Storage"....